platform/android/CHANGELOG.md

```diff
@@ -4,8 +4,11 @@ Mapbox welcomes participation and contributions from everyone.  If you'd like to
 
 ## master
 
+## 8.2.0-beta.1 - July 18, 2019
+
 ### Bugs
  - Fixed a custom geometry source bug caused by using the outdated tiles after style update [#15112](https://github.com/mapbox/mapbox-gl-native/pull/15112)
+ - Favor previous anchor only when still in the `text-variable-anchor` options [#15129](https://github.com/mapbox/mapbox-gl-native/pull/15129)
```

Comment:
Fix bug where previous anchors set in `text-variable-anchor` were still preferred after anchor options were reset